xAPI.md

```diff
@@ -237,43 +242,38 @@ JSON allows for objects to have properties that contain empty objects. This is n
 
-__Activity__: An Activity is a type of Object making up the “this” in I did “this”; it is something 
+__Activity__: An Activity is a type of Object making up the “this” in “I did this”; it is something 
 with which an Actor interacted. It can be a unit of instruction, experience, or performance that is 
 to be tracked in meaningful combination with a Verb. Interpretation of Activity is broad, meaning that 
 Activities can even be tangible objects such as a chair (real or virtual). In the statement "Anna 
 tried a cake recipe", the recipe constitutes the Activity in terms of the xAPI statement. Other 
 examples of activities include a book, an e-learning course, a hike or a meeting.
 
-<a name="def-activity-provider" />
-
-__Activity Provider (AP)__: The software object that is communicating with 
-the LRS to record information about a learning experience. May be similar to a SCORM 
-package in that it is possible to bundle learning assets with the software object that performs this 
-communication, but an Activity Provider could also be separate from the experience it is reporting about.
-
 <a name="def-actor" />
 
-__Actor__: An identity or persona of an individual or group tracked using Statements as doing an 
-action (Verb) within an Activity.
+__Actor__: An Agent (individual representation) or Group (multiple representation) tracked using Statements 
+as doing an action (Verb) within an Activity.  Is the "I" in "I did this".
 
 <a name="def-authentication" />
 
-__Authentication__: The concept of verifying the identity of a user or system. Authentication 
-allows interactions between the two "trusted" parties.
+__Authentication__: The concept of verifying identity. Authentication allows interactions between two 
+"trusted" parties.
 
 <a name="def-authorization" />
 
-__Authorization__: The affordance of permissions based on a user's or system's role; 
-the process of making one user or system "trusted" by another.
+__Authorization__: The affordance of permissions based on role; the process of making one party 
+"trusted" by another.
 
-<a name="def-baseendpoint" />
+<a name="def-baseresourcepath" />
 
-__Base Endpoint__: The maximal path under all Experience API endpoints, including a slash. E.g. an LRS with a statements endpoint of http://example.com/xAPI/statements has a Base Endpoint of http://example.com/xAPI/"
+__Base Resource Path__: The maximal path under all Experience API resources, including a slash. E.g. 
+an LRS with a Statements Resource of http://example.com/xAPI/statements has a Base Resource Path of 
+http://example.com/xAPI/"
 
 <a name="def-client" />
 
-__Client__: - Refers to any entity that might interact with an LRS. A Client can be (for example) an 
-Activity Provider, an LMS, or another LRS.
+__Client__: - Refers to any entity that might interact through requests. Some examples could be a 
+Learning Record Provider, a Learning Record Consumer, a Learning Management System, or a Learning Record Store.
```

Comment:
How about add "LMS and "LRS" as commonly used terms for people not familiar with the full names? 

"Learning Management System (LMS) or a Learning Record Store (LRS)."